tests/client.py

```diff
@@ -843,6 +945,14 @@ def create_contacts(self):
         url = f"{BASE_URL}/contacts/v1/contact"
         data = {
             "properties": [
+                {
+                    "property": "custom_string",
+                    "value": "custom_string_value"
+                },
+                {
+                    "property": "custom_number",
+                    "value": 1567
+                },
```

Comment:
Since we are setting custom properties for every call of `create_contacts` we need the `create_custom_contact_properties` to be called prior to calling the `create_contacts`.  I see you are doing this in the test, but what about other tests that would need to add contacts, those would probably break if the custom properties were not present. I believe we should call `create_custom_contact_properties` in the `__init__` of the client so that we call it once when it starts up instead of depending on the tester to know about this or remember to add it in each test.

Reply:
So, creating the custom contact property is a one time task. After my first successful run of the test, it never creates and gives a 409, that's why I am skipping it. Probably doing it in init is a good idea. Will try that

Reply:
I think you moved it to setup. I meant the init of the client so anyone using the client makes sure it is in good shape, vs just for your test.